helm/reana/templates/secrets.yaml

```diff
@@ -23,6 +23,19 @@ data:
   CERN_CONSUMER_KEY: {{ .Values.secrets.cern.sso.CERN_CONSUMER_KEY | default "cern_consumer_key" | b64enc }}
   CERN_CONSUMER_SECRET: {{ .Values.secrets.cern.sso.CERN_CONSUMER_SECRET | default "cern_consumer_secret" | b64enc }}
 ---
+{{- if .Values.login }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "reana.prefix" . }}-login-provider-secrets
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/resource-policy": keep
+type: Opaque
+data:
+  PROVIDER_SECRETS: {{ .Values.secrets.login | toJson | b64enc }}
```

Comment:
as before, let's call this `LOGIN_PROVIDERS_SECRETS` for consistency with the rest?